libraries/botbuilder-schema/botbuilder/schema/teams/_models_py3.py

```diff
@@ -2058,3 +2086,265 @@ def __init__(
+        self.state = state
+        self._custom_init()
+
+    def _custom_init(self):
+        return
+
+
+class TabResponseCard(Model):
+    """
+    Envelope for cards for a Tab request.
+
+    :param card: Gets or sets adaptive card for this card tab response.
+    :type card: object
+    """
+
+    _attribute_map = {
+        "card": {"key": "card", "type": "object"},
+    }
+
+    def __init__(self, *, card=None, **kwargs) -> None:
+        super(TabResponseCard, self).__init__(**kwargs)
+        self.card = card
+        self._custom_init()
+
+    def _custom_init(self):
+        return
+
+
+class TabResponseCards(Model):
+    """
+    Envelope for cards for a TabResponse.
+
+    :param cards: Gets or sets adaptive card for this card tab response.
+    :type cards: list[object]
+    """
+
+    _attribute_map = {
+        "cards": {"key": "cards", "type": "[object]"},
+    }
+
+    def __init__(self, *, cards=None, **kwargs) -> None:
+        super(TabResponseCards, self).__init__(**kwargs)
+        self.cards = cards
+        self._custom_init()
+
+    def _custom_init(self):
+        return
+
+
+class TabResponsePayload(Model):
+    """
+    Initializes a new instance of the TabResponsePayload class.
+
+    :param type: Gets or sets choice of action options when responding to the
+     tab/fetch message. Possible values include: 'continue', 'auth' or 'silentAuth'
+    :type type: str
+    :param value: Gets or sets the TabResponseCards when responding to 
+     tab/fetch activity with type of 'continue'.
+    :type value: TabResponseCards
+    :param suggested_actions: Gets or sets the Suggested Actions for this card tab.
+    :type suggested_actions: TabSuggestedActions
+    """
+
+    _attribute_map = {
+        "type": {"key": "type", "type": "str"},
+        "value": {"key": "value", "type": "TabResponseCards"},
+        "suggested_actions": {"key": "suggestedActions", "type": "TabSuggestedActions"},
+    }
+
+    def __init__(
+        self, *, type=None, value=None, suggested_actions=None, **kwargs
+    ) -> None:
+        super(TabResponsePayload, self).__init__(**kwargs)
+        self.type = type
+        self.value = value
+        self.suggested_actions = suggested_actions
+        self._custom_init()
+
+    def _custom_init(self):
+        return
+
+
+class TabSumit(Model):
+    """
+    Invoke ('tab/submit') request value payload.
+
+    :param tab_entity_context: Gets or sets current tab entity request context.
+    :type tab_entity_context:
+     ~botframework.connector.teams.models.TabEntityContext
+    :param context: Gets or sets current tab entity request context.
+    :type context:
+     ~botframework.connector.teams.models.TabContext
+    :param data: User input data. Free payload containing properties of key-value pairs.
+    :type data:
+     ~botframework.connector.teams.models.TabSubmitData
+    """
+
+    _attribute_map = {
+        "tab_entity_context": {"key": "tabEntityContext", "type": "TabEntityContext"},
```

Comment:
I believe we need [the same `tabContext` change](https://github.com/microsoft/botbuilder-dotnet/blob/cb33e75b72735e25177fb82084813dcaeadf16c7/libraries/Microsoft.Bot.Schema/Teams/TabSubmit.cs#L27) here.